src/wireviz/DataClasses.py

```diff
@@ -7,6 +7,16 @@
 from wireviz import wv_colors
 
 
+@dataclass
+class Image:
+    src: str
+    scale: str = "false"
```

Comment:
Please add a comment to show the possible values... no explanation (that's something for the `docs`), just a simple cheat sheet:
```python
    scale: str = "false"  #  false | true | width | height | both
```

Reply:
And probably make the attribute `Optional`... it's not used in `ex08.yml`, for example

Reply:
> And probably make the attribute `Optional`... it's not used in `ex08.yml`, for example

It is optional, but I just followed the pattern from the other optional bool attributes:
```
    show_name: bool = None
    show_pincount: bool = None
    hide_disconnected_pins: bool = False
    autogenerate: bool = False
```

Reply:
TBH I am not sure what effect the `Optional[]` keyword has.
Is there a difference between the following two lines?
```python
    scale: str = 'false'
    scale: Optional[str] = 'false'
```
I'm having trouble finding an answer, and currently both versions are used in `DataClasses.py` so I would like to decide on one for everything :)